### PR TITLE
Last minute updates

### PR DESCRIPTION
**Pull request by miskreant**
- Bugfix #116 from origin
- Modified to identify upper & lower range handles

Tested slider on master

### CODE REVIEW

File: src/Slider.jsx [hunk 1]
```diff
@@ -270,7 +270,7 @@ class Slider extends React.Component {
 
     const points = Object.keys(marks).map(parseFloat);
     if (step !== null) {
-      const closestStep = Math.round(val / step) * step;
+      const closestStep = (Math.round((val - min) / step) * step) + min;
```
Comment (wcjordan) [inline src/Slider.jsx:273]: Have we made a PR on the main repo with these changes?  Can you link here and take charge of making sure that's merged in when the main repo is updated?

Comment (miskreant) [inline src/Slider.jsx:273]: This change came directly from the origin repo: [react-component/slider](https://github.com/react-component/slider/commit/e2504d99fe845d686ac468279b5df5008cc8ed04) so it should be good to go. Unless you're asking me to submit a PR to react-component for the upper/lower handle name stuff (which they may or may not accept).

Comment (wcjordan) [inline src/Slider.jsx:273]: Ah okay, this ones fine.  

We should submit a PR for the classnames part, potentially taking their suggestion for fixing if they don't want the PR.  We don't want to end up maintaining our own fork of the component.

**Comment by wcjordan** [comment]
lgtm if we follow through on PR and merging back to their fork

File: src/Slider.jsx (hunk 1 above)
Comment (miskreant) [inline src/Slider.jsx:273]: Sounds good, I'll take care of it.

**Pull request merged**